app/assets/javascripts/transactions.js

```diff
@@ -24,6 +24,10 @@ $(function () {
     e.stopPropagation();
   });
 
+  $(document).on('click', '.account-link', function(e) {
```

Comment:
@Gataniel let's change it to $(document).on('click', '[data-stop-propagation='true']', function(e) {``.